CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ option(USE_OPENGL "Enable libraries that use OpenGL" ON)
 option(USE_HDF5 "Enable optional HDF5 features" OFF)
 option(USE_QT "Enable libraries that use Qt 5" ON)
 option(USE_VTK "Enable libraries that use VTK" OFF)
+option(USE_LIBARCHIVE "Enable optional Libarchive features" ON)
```

Comment:
I don't see how the use of libarchive is actually optional at this point, I think we should make it optional.